src/main/fc/fc_core.c

```diff
@@ -397,6 +397,16 @@ void processRx(timeUs_t currentTimeUs)
     static bool armedBeeperOn = false;
     static bool airmodeIsActivated;
 
+    if (rxRuntimeConfig.processingRequired) {
+        rxRuntimeConfig.processingRequired = !rxRuntimeConfig.rcProcessFrameFn(&rxRuntimeConfig);
```

Comment:
I think it would be better to call `rcProcessFrameFn` from within `calculateRxChannelsAndUpdateFailsafe` rather than here.

Reply:
Good point re `calculateRxChannelsAndUpdateFailsafe`.